updatecli/README.adoc

```diff
@@ -0,0 +1,98 @@
+= README
+
+link:https://img.shields.io/github/actions/workflow/status/kubewarden/automation/validate-updatecli.yaml?branch=main[image:https://img.shields.io/github/actions/workflow/status/kubewarden/automation/validate-updatecli.yaml?branch=main[GitHub Workflow Status]]
+
+
+This repository contains a list of common Updatecli published on ghcr.io/kubewarden/updatecli/**
+
+== HOWTO
+
+**Login**
+
+Even though all Updatecli policies published on `ghcr.io` are meant to be public, you'll probably need to authenticate to reduce rate limiting by running:
+
+    docker login ghcr.io
+
+**Publish**
+
+Each of the policies defined in this repository are automatically published on ghcr.io via a GitHub Action workflow
+
+**Show**
+
+We can see the content of the policy by running:
+
+    updatecli manifest show ghcr.io/kubewarden/updatecli/<a policy name>:latest
+
+**Use**
+
+They are two ways to execute an Updatecli policy, either running one policy or several policies at once.
+
+One policy can be executed by running:
+
+    updatecli apply --config ghcr.io/kubewarden/updatecli/<a policy name>:latest
+
+
+IMPORTANT: Any values files specified at runtime will override default values setting from the policy bundle
+
+Assuming we have a file named `update-compose.yaml`, multiple policies can be composed and executed by running:
+
+        updatecli compose apply
+
+.update-compose.yaml
+```yaml
+policies:
+    - policy: "ghcr.io/kubewarden/updatecli/autodiscovery/golang:latest"
```

Comment:
```suggestion
    - policy: "ghcr.io/kubewarden/automation/open-release-pr:latest"
```

Reply:
I'm skipping applying this, the next line would also be wrong nevertheless.